backend/app/models/project.rb

```diff
@@ -39,7 +39,8 @@ class Project < ApplicationRecord
            dependent: :destroy
 
   ## Validations
-  validates :title,       presence: true
+  validates :title,       presence: true,
```

Comment:
This is a good idea! What do you think about preventing an empty string from being accepted? I'm actually not 100% sure it's necessary to do this.

```suggestion
before_validation { title&.strip! }
validates :title, presence: true, length: { maximum: 255 }

```